test/benchmarks/png_funcs.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
```

Comment:
Maybe we can rename this file to `benchmark_png_shared.h`? Thoughts?

Reply:
Sure